source/_patterns/02-organisms/22-settings/01-auto-deposit-settings.mustache

```diff
@@ -1,13 +1,26 @@
 {{#is-standalone}}
-	<form data-abide="ajax">
+	<form data-abide="ajax" class="settingsForm xbLegacyNav auto-deposit-form">
 {{/is-standalone}}
 
-<h3 class="formTitle">Auto deposits</h3>
+	<h3 class="formTitle">Monthly Good Subscription</h3>
+
+	{{^is_subscriber}}
+		<p>
+			<a href="{{URL_TO_MONTHLY_GOOD}}">Sign up for a Kiva Monthly Good subscription</a>
+		</p>
+	{{/is_subscriber}}
+
+{{^is_subscriber}}
+	<h3 class="formTitle">Auto deposits</h3>
+{{/is_subscriber}}
+
 {{^has_auto_deposit}}
 	<p>
 		<a href="{{setup_url}}">Add money to your account every month</a>
```

Comment:
question for Brandon - if the MG prompt is also showing, should this prompt include the Auto deposit lnguage, like "Auto deposit - add money to your account every month"?